src/logger.rs

```diff
@@ -1,33 +1,31 @@
-use fern;
 pub use log::LevelFilter;
-use std::env;
-use std::io;
+
+use fern;
+use std::{env, io, path::PathBuf, str::FromStr};
 
 /// Logger configuration object.
-#[derive(Clone, Copy)]
+#[derive(Clone)]
 pub struct LoggerConfig {
+    /// Determines whether to log to the terminal or not.
+    pub use_stdout: bool,
     /// Whether to use color output when logging to the terminal or not.
     pub use_colors: bool,
```

Comment:
Perhaps the two variables above should be combined into an enum since `use_colors` depends on `use_stdout`.